css/main.css

```diff
@@ -0,0 +1,148 @@
+body {
+	color: white !important;
+	font-size: 16px;
```

Comment:
Jeśli dobrze rozumiem, to robimy reset CSSa (reset.css), żeby go potem znormalizować (początek w3.css; po resecie ma to ograniczony sens), żeby potem ustalić standardowe rozmiary (główna część w3.css; nie wiem po co nam reset i normalizacja skoro w3.css i tak wszystko ustala), po czym je poprawić (tutaj; czyli wybraliśmy framework W3, a później i tak jesteśmy mądrzejsi). Trochę to nadmiarowe. Zamiast tego proponowałbym jedną z dwóch strategii:
1. Po prostu opieramy się na wbudowanych stylach każdej przeglądarki i zmieniamy to co nam potrzebne.
2. Robimy normalizację wbudowanych stylów przeglądarki, a później dostosowujemy to co nam nie odpowiada.

Reset i budowanie stylu od zera wydaje mi się bez sensu w dzisiejszych czasach.

Reply:
Nie jestem pewien, czy dobrze rozumiem.
Moje intencje to dokładnie opcja 2), wbudowane style niektórych przeglądarek są po prostu brzydkie ;)

reset.css normalizuje o wiele więcej własności niż później ustalamy w w3.css czy main.css (np. marginesy i wyłożenie). Fakt, reset.css stworzony jest głównie z myślą o mniej standardowych i nieco nieprzewidywalnych przeglądarkach (głównie na urządzenia mobilne) i mi osobiście daje niejako spokój ducha :)

Rozdzielenie stylów odnoszących się do tych samych elementów pomiędzy trzy różne pliki odzwierciedla logiczny podział według tego co te reguły mają na celu (reset - normalizacja, w3 - ogólny framework i responsywność, main - "skórka" i cała reszta rzeczy specyficznych dla tego projektu. Idąc dalej za tą logiką kusiło mnie nawet żeby rozdzielić "skórkę" i całą resztę, ale się powstrzymałem. ;)

Reply:
Jeśli robimy normalizację to nie wiem po co reset, tu jest dobry artykuł na ten temat: https://pineco.de/a-brief-about-css-reset-and-normalize/

Nie wiem czy dla naszego projektu normalizacja jest konieczna, ale jeśli tak to może po prostu użyć czegoś w miarę małego i świeżego jak https://github.com/csstools/normalize.css ?

W3CSS to fajny framework, ale my go tutaj prawie nie używamy. Wykorzystujemy normalizację, którą można załatwić osobno i parę elementów, które i tak są sensowne po normalizacji. Te szczegóły których potrzebujemy spokojnie można zintegrować w skórce i moim zdaniem będzie to bardziej czytelne. Np. nikt nie będzie musiał pamiętać by używać paru magicznych klas na wybranych elementach.